### PR TITLE
[TMVA] configure tutorials/tmva/makefile

### PR DESCRIPTION
**Pull request by pseyfert**
example makefile from tutorials/tmva should use the same c++ standard as
root (otherwise one faces errors from string views in the root headers
when building the tutorials).

Note:
 - I also changed in the global CMakeLists.txt `tutorial_file` into `artifact_file`, (which looks like a renaming variable mistake to me, but didn't go through the history)
 - This causes `tutorials/tmva/makefile` to not appear in artifact_files_builddir and thus the dependencies of move_artifacts. → I am not sure if I broke dependencies here.

(WIP struggeling to install the configured makefile)

### CODE REVIEW

File: tutorials/tmva/makefile.in [hunk 1]
```diff
@@ -16,7 +16,7 @@ BINS = TMVAClassification \
 
 
 CXX = g++
-CCFLAGS = -O0 -ggdb -std=c++11 
+CCFLAGS = @CMAKE_CXX_FLAGS@ -O0 -ggdb
```
Comment (amadio) [inline tutorials/tmva/makefile.in]: Can't this use `root-config --cxxflags`?

**Comment by amadio** [comment]
@phsft-bot build

**Comment by lmoneta** [comment]
For me this PR is ready to be merged. What do you think @amadio ?

**Comment by amadio** [comment]
Seem ok for me too, just squash, please, since some commits are fixups that should not be in the history.

**Comment by amadio** [comment]
Actually, this is assigned to me, so I will merge.

**Pull request merged**